src/Evolution/ComputeTags.hpp

```diff
@@ -21,12 +21,9 @@ namespace Tags {
  */
 template <size_t Dim, typename AnalyticSolutionTag,
           typename AnalyticFieldsTagList>
-struct AnalyticCompute
-    : db::add_tag_prefix<::Tags::Analytic,
-                         ::Tags::Variables<AnalyticFieldsTagList>>,
-      db::ComputeTag {
-  using base = db::add_tag_prefix<::Tags::Analytic,
-                                  ::Tags::Variables<AnalyticFieldsTagList>>;
+struct AnalyticCompute : ::Tags::AnalyticSolutions<AnalyticFieldsTagList>,
+                         db::ComputeTag {
+  using base = ::Tags::AnalyticSolutions<AnalyticFieldsTagList>;
```

Comment:
Just so I understand the plan: this will ultimately return a `std::optional<Variables>`, grabbing `Tags::AnalyticSolutionOrData` (or whatever it's called), checking if the thing is an analytic solution or not, and then set the `std::optional<Variables>` appropriately. Is that correct?

Reply:
Yep!